;

```diff
@@ -0,0 +1,547 @@
+// Copyright (c) 2013-2017 The btcsuite developers
```

Comment:
I think you added this file as a mistake.

Reply:
Indeed...removed